src/test/java/Problem2Test.java

```diff
@@ -7,13 +7,21 @@
     public void testRemoveDuplicates() {
         int[][] inputs = {
                 // DO NOT REMOVE OR CHANGE THE GIVEN TEST CASES!
-                {0, 0, 1, 1, 1, 2, 2, 4, 4, 6}
-                // add more test cases
+                {0, 0, 1, 1, 1, 2, 2, 4, 4, 6},
+                {0, 0, 0, 1, 1, 2, 2, 3, 4, 5, 5},
+                {1, 1, 1, 2, 2, 2, 3, 4, 4, 5, 5},
+                {1, 3, 3, 6, 6, 7, 7, 8, 9, 9, 9},
+                {2, 2, 3, 5, 5, 5, 9, 9, 9},
+
         };
 
         int[][] expects = {
-                {0, 1, 2, 4, 6}
-                // add more test cases
+                {0, 1, 2, 4, 6},
+                {0, 1, 2, 3, 4, 5},
+                {1, 2, 3, 4, 5},
+                {1, 3, 6, 7, 8, 9},
+                {2, 3, 5, 9}
```

Comment:
great test. let's add more cases for the edge cases, such as null array and empty array.